dotnetManual/antora/antora.yml

```diff
@@ -0,0 +1,15 @@
+name: dotnet-manual
+title: Neo4j Dotnet Manual
```

Comment:
do these titles surface in the manual?

if so, I think it should be **The Neo4j .NET Driver Manual**

Reply:
Good point. This isn't an attribute that would be used to generate title text from our asciidoc, but it is potentially used by Antora in a few places (for example, if we used the version picker to display the list of available manuals as well as versions of those manuals, this title attribute would be displayed in that list.

Reply:
Also in the browser tab - good spot!

Reply:
All title attributes updated to match the manual's index page title